scripts/externalize_js.sh

```diff
@@ -122,13 +122,27 @@ if [[ $autocommit == "true" ]]; then
 fi
 
 
+variable_open_bracket_regex="{\(%\|{\)"
+variable_close_bracket_regex="\(%\|}\)}"
 # check where in page there are template tags
-template_tags=`sed -n "/{%/=; /{{/=" $new_module_location`
+template_tags=`sed -n "/$variable_open_bracket_regex/=" $new_module_location`
 template_tag_count=`echo $template_tags | wc -l`
 if [ "$template_tag_count" -gt 0 ]; then
+
+    gettext_open="gettext("
+    gettext_close=")"
+    sed -i "/\('\|\"\)$variable_open_bracket_regex trans/s/ $variable_close_bracket_regex\(\"\|'\)/$gettext_close/; s/\('\|\"\)$variable_open_bracket_regex trans /$gettext_open/" $new_module_location
```

Comment:
Big fan of the attempt to make `sed` expressions at all readable.  You could probably split this on to multiple lines too, although I'm not 100% sure that won't break something.  Also, it's customary to use upper-case letters for bash variables which may help distinguish them from the non-variable parts of this.  You could also make a variable `$QUOTE`, so this would look like:
```$QUOTE$OPEN_BRACKET trans/s/...```

Reply:
Ha yeah, making them less opaque is definitely a goal. Adding these suggestions ^^

Reply:
In my editor, the variables change color so it isn't too difficult to distinguish, but the caps is helpful as well in case I ever don't have that